_includes/pbc/all/install-features/202212.0/marketplace/install-the-marketplace-merchant-portal-core-feature.md

```diff
@@ -604,8 +604,8 @@ Follow the steps below to install the Merchant Portal Core feature frontend.
 ### Prerequisites
 
 Environment requirements:
-- NPM v6 (higher versions have problems with workspace)
-- NodeJs v12-14
+- npm v6 (higher versions have problems with workspace)
+- Node.js v12-14
```

Comment:
This version must already use node 16 and npm 8.
The previous version should be added for 202204.0